### ISSUE TITLE
markdown_py executable line endings (version 2.4)

### ISSUE THREAD

**Issue by samueljohn** [Python-Markdown/markdown#303]
On OS X 10.9, after `pip install markdown`, I have version 2.4 installed with my brewed python 2.7.6 in zsh:

The executable script seems to contain a `\r` character at the end of the shebang line which results in this error, I think:

```
markdown_py
env: python\r: No such file or directory
```

After converting the line ending to unix style in `markdown_py`, it works again.

**Comment by samueljohn** [comment]
Note, `which python` works and is found in `PATH`.

**Comment by waylan** [comment]
Yes, I'm aware of the issue. See #289 for an explanation of what happened.  Unfortunately, I'm still unpacking from a move so I still don't have access to the proper machine to do a proper bugfix release. Once I'm set back up, I'll issue a new release which fixes this.

In the meantime, I would suggest using python's `-m` option:

```
python -m markdown
```

Actually the [docs](https://pythonhosted.org/Markdown/cli.html) recommend this over the markdown_py script. It's what I use. The only reasons the script still exists is that we still support Python 2.6 which has only limited support for it, and for all those people who have existing scripts that might still be using the old script.

**Comment by samueljohn** [comment]
Thanks. I see. I just saw the executable has been installed and used that.

**Comment by waylan** [comment]
Just released [2.4.1](https://pypi.python.org/pypi/Markdown/2.4.1) which fixes this.
